sql-to-dbsp-compiler/SQL-compiler/src/test/resources/metadataTests-generateDF.json

```diff
@@ -76,21 +76,24 @@
       "calcite": {
         "partial": 0
       },
-      "positions": []
+      "positions": [],
+      "merkle": "8b384059bdb44ad811ab341cc5e2a59697f39aac7b463cab027b185db8105e73"
```

Comment:
Kind of nitpicking, but `merkle` is probably not a good name for this. It doesn't matter what algorithm was used to generate the id, the important part is that it's a persistent id of the operator, so `persistent_id` might be a better name.